lib/ControlledVocab/tests/ControlledVocab-test.js

```diff
@@ -6,118 +6,156 @@ import {
   it,
 } from '@bigtest/mocha';
 
+import React from 'react';
 import ControlledVocabInteractor from './interactor';
 import mountComponent from './mountComponent';
 
 import { setupApplication } from '../../../tests/helpers';
 
+import ControlledVocab from '../ControlledVocab';
 
-describe('ControlledVocab', () => {
+describe.only('ControlledVocab', () => {
   const cv = new ControlledVocabInteractor();
 
-  setupApplication();
+  describe('User can edit EditableListForm', () => {
+    setupApplication();
 
-  beforeEach(mountComponent);
+    beforeEach(() => mountComponent(true, server));
 
-  describe('clicking New button', () => {
-    beforeEach(async () => {
-      await cv.newButton().click();
-    });
+    describe('clicking New button', () => {
+      beforeEach(async () => {
+        await cv.newButton().click();
+      });
 
-    it('should disable New button', () => {
-      expect(cv.disabledNewButton).to.be.true;
-    });
+      it('should disable New button', () => {
+        expect(cv.disabledNewButton).to.be.true;
+      });
 
-    it('should display input field', () => {
-      expect(cv.inputField).to.exist;
-    });
+      it('should display input field', () => {
+        expect(cv.inputField).to.exist;
+      });
 
-    it('should disable Save button', () => {
-      expect(cv.disabledSaveButton).to.be.true;
-    });
+      it('should disable Save button', () => {
+        expect(cv.disabledSaveButton).to.be.true;
+      });
 
-    it('should display Cancel button', () => {
-      expect(cv.cancelButton).to.exist;
+      it('should display Cancel button', () => {
+        expect(cv.cancelButton).to.exist;
+      });
+
+      describe('clicking Cancel button', () => {
+        beforeEach(async () => {
+          await cv.cancelButton().click();
+        });
+
+        it('should enable New button', () => {
+          expect(cv.disabledNewButton).to.be.false;
+        });
+      });
     });
 
-    describe('clicking Cancel button', () => {
+    describe('clicking New button', () => {
       beforeEach(async () => {
-        await cv.cancelButton().click();
+        await cv.newButton().click();
       });
 
-      it('should enable New button', () => {
-        expect(cv.disabledNewButton).to.be.false;
+      describe('when blurring the input field', () => {
+        beforeEach(async () => {
+          await cv.blurInputField();
+        });
+
+        it('should display the error message', () => {
+          expect(cv.emptyFieldError).to.equal('Please fill this in to continue');
+        });
       });
-    });
-  });
 
-  describe('clicking New button', () => {
-    beforeEach(async () => {
-      await cv.newButton().click();
+      describe('when focusing on the input field', () => {
+        beforeEach(async () => {
+          await cv.fillInputField('asdf');
```

Comment:
Please use proper values for inputs. Of course, it doesn't matter in the end but I don't think that input should be filled with 'asdf'.